### DISCUSSION TITLE
Server admin unable to migrate followers to a new server

### DISCUSSION THREAD

**Discussion by kallisti5** [mastodon/mastodon#22357] [Q&A]
### Steps to reproduce the problem

1. Server admin over a single user instance of mastodon
2. Attempt to migrate to a new server (mastondon.sdf.org in this instance)
3. Do the account alias on mastodon.sdf.org
4. Do the migration on my server
5. While my account is redirecting to the new server's account... my followers are still referencing my old account and don't migrate.

### Expected behaviour

Followers migrate

### Actual behaviour

They remain following my old account

### Specifications

Mastodon 4.0.2 to 4.0.2  .  Also attempted on 3.5.2 -> 4.0.2

**Comment by nightpool** [comment]
I'm sorry, you'll have to ask the mastodon.sdf admins for help here, there's nothing we can do, we just write the software, we don't run it.

**Comment by kallisti5** [comment]
You can find the old account (with all the followers) here: https://ukiyo.cafe/@kallisti5
The target account has 0 followers.

**Comment by kallisti5** [comment]
@nightpool as the admin of the source server.. is there anything I can look at in the server logs?

**Comment by nightpool** [comment]
Oh, my mistake, you're migrating TO sdf, away from your single user instance. In that case I'll convert this to a debugging discussion

**Comment by kallisti5** [comment]
No worries.   I was assuming it's because i'm the sole admin (and user) of my instance and mastodon doesn't like migrating the "last user" away

**Reply by nightpool** [reply]
No, that shouldn't affect things at all. I think it's more likely that maybe you're having problems with your sidekiq configuration or your network activity and the Move activities aren't going through

**Reply by kallisti5** [reply]
![sidekiq_1](https://user-images.githubusercontent.com/100110/208028109-3d3053b9-cdf6-4172-804c-f481c93a51d3.png)

**Reply by kallisti5** [reply]
For retry queues...  some ActivityPub::DeliveryWorker retries to servers (not mastodon.sdf), AdminMailer, and ActivityPub::SynchronizeFeaturedCollectionWorker to an unrelated server
